eidas_node/saml.py

```diff
@@ -464,7 +464,7 @@ def verify_assertion(self, cert_file: str) -> bool:
         self._verify_and_remove_signature(self.assertion_signature, cert_file)
         return True
 
-    def create_light_response(self) -> LightResponse:
+    def create_light_response(self, auth_class_map: Optional[Dict[str, LevelOfAssurance]] = None) -> LightResponse:
```

Comment:
When you declare the type of argument and it's default value is `None`, it's redundant to mark it as `Optional`. `mypy` will derive this automatically.

Reply:
Fixed.